### DISCUSSION TITLE
Can someone please verify if Jedis 4.4.3 supports Redis Engine 4.0.10?

### DISCUSSION THREAD

**Discussion by handvo** [xetorthio/jedis#3494] [Q&A] [answered]
I have found these in the docs, but I need something more concrete for planning. Thank you in advance.

- "The most recent version of this library supports redis version 5.0, 6.0, 6.2, and 7.0." which is Jedis 5.0. 
- "In another doc "Jedis is fully compatible with redis 2.8.x, 3.x.x and above*"

Does that mean that the current Jedis version should also support any Redis from 2.8.x and above?

**Comment by sazzad16** [comment]
[NOT AN ANSWER]

Could you please give the link of that another doc saying "Jedis is fully compatible with redis 2.8.x, 3.x.x and above*"?

Thanks!

**Reply by handvo** [reply]
https://cljdoc.org/d/redis.clients/jedis/1.5.3-SNAPSHOT/doc/readme Could be very old I think.... But the version in this doc (3.8.0) is the one I am using

**Comment by sazzad16** [comment, marked as answer]
It should.

But it's not officially tested.
